web/styles/right_sidebar.css

```diff
@@ -261,6 +261,15 @@ $user_status_emoji_width: 24px;
     opacity: var(--opacity-sidebar-heading);
     transition: opacity 140ms linear;
     grid-area: row-content;
+    display: grid;
+    grid-template-columns: minmax(0, 1fr) auto auto auto;
+}
+
+.buddy-list-heading-text {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    margin-right: 2px;
 }
```

Comment:
this change seems bugged.

![image](https://github.com/user-attachments/assets/675c654a-1a8c-4524-b29a-87cfd31a4f0c)

Reply:
Fixed by switching to flex

<img width="248" alt="image" src="https://github.com/user-attachments/assets/5abf3345-e518-4eb7-a3f1-7f8550bedc31">